<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLTTestRobotPluginForm</class>
 <widget class="QWidget" name="DLTTestRobotPluginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLT Test Robot Plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxRunAllTest">
     <property name="text">
      <string>Run all Tests</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Tests Filename:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditTestsFilename"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonTestFilename">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Current Test Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Repeat:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QSpinBox" name="spinBoxRepeat">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButtonStartTests">
     <property name="text">
      <string>Start Tests</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QComboBox" name="comboBoxCurrentTestName"/>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButtonStopTests">
     <property name="text">
      <string>Stop Tests</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Test Status:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Commands:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditAllRepeatNo"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditCurrenTest"/>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QListWidget" name="listWidgetCommands"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditTestRepeatNo"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditCmdNo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>All Repeat No.:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Current Test:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Cmd No.:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Test Repeat No.:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Tests Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditTestsStatus"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Failed Test Commands:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditFailedTestCommands"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
